lib/JITIDL/ChakraJIT.idl

```diff
@@ -3,8 +3,13 @@
 // Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
 //-------------------------------------------------------------------------------------------------------
 
+cpp_quote("#ifndef __JITTypes_h__")
+cpp_quote("#define __JITTypes_h__")
```

Comment:
previously Linux use JITTypes.h, windows uses the midl compiled result. without this change, every annotation in this file won't work on windows
with this change, both Linux and windows will use JITTypes.h. while RPC marshaling, it's using the compile result(chakraJIT.h), which does not contain any annotation.

---
In reply to: [92255395](https://github.com/Microsoft/ChakraCore/pull/2191#discussion_r92255395) [](ancestors = 92255395)